### PR TITLE
Logout endpoint' front-end side related to #8 #9

### PR DESCRIPTION
**Pull request by ameentalahmeh**
- [X] using `axios.get` for request logout endpoint.   
- [X] Then, Add a `handleSubmit` function for handling the `click` on the logout button, using the props that passed from the other components.

### CODE REVIEW

File: client/src/components/NavBar/index.js [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
+
+function handleSubmit(props){
+
+  const { history } = props;
+  axios
+    .get("/logout")
+    .then(() => {
+      history.push("/");
+    })
+    .catch(error => console.log(error));
```
Comment (shahenazmonia) [inline client/src/components/NavBar/index.js]: Render ```error``` by redirect the user to Error page

File: server/controllers/checkAuth.js [hunk 2]
```diff
@@ -1,16 +1,2 @@
...
-    const token = cookie.parse(req.headers.cookie).logged_in;
-    jwt.verify(token, SECRET, (err) => {
-      if (err) {
-        return res.status(401).json({ error: 'something went wrong!' });
-      }
-      return res.json({ success: true });
-    });
-  }
-};
+exports.checkAuth = (req, res) => res.json({ success: true });
```
Comment (shahenazmonia) [inline server/controllers/checkAuth.js]: Why did you push this file here! I merged the file to master!

Comment (ameentalahmeh) [reply server/controllers/checkAuth.js]: بالغلط :100:

File: server/router.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ const { auth } = require('./middlewares/auth.js');
 const router = express.Router();
 
 router.get('/events', auth, getEvents);
-router.get('/checkauth', checkAuth);
+router.get('/checkauth', auth, checkAuth);
```
Comment (shahenazmonia) [inline server/router.js]: same thing here, this file was in the previous PR, I merged it to master before!

Comment (ameentalahmeh) [reply server/router.js]: done :+1:

File: client/src/components/NavBar/index.js (hunk 1 above)
Comment (ameentalahmeh) [reply client/src/components/NavBar/index.js]: done :+1:

**Comment by shahenazmonia** [comment]
@ameentalahmeh Branch has conflict with master, fix it and push the code again

**Comment by ameentalahmeh** [comment]
@shahenazmonia , can you check again :+1:

File: client/src/components/NavBar/index.js [hunk 4]
```diff
@@ -1,12 +1,28 @@
...
+function handleSubmit(props){
+
+  const { history } = props;
+  axios
+    .get("/logout")
+    .then(() => {
+      history.push("/");
+    })
+    .catch(error => {
+      const { history } = this.props;
```
Comment (shahenazmonia) [inline client/src/components/NavBar/index.js]: You defined ```history``` at the beginning of the function, you don't have to define it again

**Comment by ameentalahmeh** [comment]
done, it removed :+1:

**Pull request merged**